Libraries/Utilities/MatrixMath.js

```diff
@@ -581,7 +581,7 @@ var MatrixMath = {
       skew,
       translation,
 
-      rotate: rotationDegrees[2],
+      perspectiveZ: perspective[2] != 0 ? -1/perspective[2] : 1280,
```

Comment:
we should keep it just ``perspective``, just like ios. I see some perspective code already in decomposed matrix. you can pass perspective from js like:

```
transform: { perspective: 1280 }
```